appveyor.yml

```diff
@@ -3,7 +3,7 @@ os: Visual Studio 2015
 environment:
   matrix:
     - nodejs_version: 0.10.40
-    - nodejs_version: 0.12.7
+    - nodejs_version: 4.4.2
   MapboxAccessToken:
```

Comment:
Okay to just remove both v12 and v4 testing on appveyor for now. I'm still working through getting node binaries for v4 working on windows: https://github.com/mapbox/node-cpp11/issues/26